lib/index.js

```diff
@@ -51,7 +51,18 @@ Comscore.prototype.loaded = function() {
  * @param {Object} page
  */
 
-Comscore.prototype.page = function() {
+Comscore.prototype.page = function(page) {
   var comScoreOptions = { c1: this.options.c1, c2: this.options.c2 };
+  var beaconParamMap = this.options.beaconParamMap;
+  var properties = page.properties();
+
+  Object.keys(this.options.beaconParamMap).forEach(function(property) {
+    if (property in properties) {
+      var key = beaconParamMap[property];
+      var value = properties[property];
+      comScoreOptions[key] = value;
```

Comment:
Does this permit someone to override c1 and c2 and, if so, can this cause Bad Things™? If not, then all good!

Reply:
It would. I'll change the PR to ignore their mapping of `c1` and `c2` by changing the order stuff is assigned.